runtime/development/src/migrations.rs

```diff
@@ -49,6 +53,19 @@ pub type UpgradeDevelopment1047 = (
 		CollatorReward,
 		AnnualTreasuryInflationPercent,
 	>,
+	// Initialize OpenGov Technical Committee with Alice
+	runtime_common::migrations::technical_comittee::InitMigration<crate::Runtime, InitialTcMembers>,
+	runtime_common::migrations::increase_storage_version::Migration<crate::Referenda, 0, 1>,
```

Comment:
Just curious, a pallet just added to the runtime, does not contain the latest version?

Reply:
Apparently! I have to admit that I did not dig deeper into this but `try-runtime` complained about it. It really surprised me!